tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal

```diff
@@ -155,10 +155,55 @@ type Address record {
     never country?;
 };
 
+type Employee record {|
+    int id;
+    string name;
+    string dept;
+|};
+
+
 function testSpreadFieldWithRecordTypeHavingNeverField() {
     Grades grades = { physics: 75 };
     Address address= { street: "Main Street" };
 
+    record {| string street; |} addressInline = {...address};
```

Comment:
Not related to your change, but this should have failed anyway since `Address` is an open record - https://github.com/ballerina-platform/ballerina-lang/issues/41740. 

Shall we fix this test so that it continues to pass even when the other bug is fixed?

Reply:
Changed the Address record into a closed record